rust/src/participant.rs

```diff
@@ -221,29 +248,29 @@ mod tests {
     #[test]
     fn test_check_task() {
         let mut part = Participant::new().unwrap();
-        let elligible_signature = Signature::from_slice_unchecked(&[
-            229, 191, 74, 163, 113, 6, 242, 191, 255, 225, 40, 89, 210, 94, 25, 50, 44, 129, 155,
-            241, 99, 64, 25, 212, 157, 235, 102, 95, 115, 18, 158, 115, 253, 136, 178, 223, 4, 47,
-            54, 162, 236, 78, 126, 114, 205, 217, 250, 163, 223, 149, 31, 65, 179, 179, 60, 64, 34,
-            1, 78, 245, 1, 50, 165, 47,
+        let eligible_signature = Signature::from_slice_unchecked(&[
```

Comment:
Why was this changed? Also, would it be possible to add a comment to explain how these values were found?

Reply:
i changed it here to make it consistent with the changed tests in `utils.rs`, other than that there was no reason.
the values were found by generating a random key pair, calculating the corresponding signature and checking by hand if the eligibility criterium was fulfilled.